### PR TITLE
Final version of Frontend

### PR DESCRIPTION
**Pull request by CadarIustin**

### CODE REVIEW

File: app/src/main/res/layout/admin_panel.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="5dp"
+            android:fontFamily="@font/share_regular"
+            android:shadowColor="@color/black"
+            android:shadowDx="6"
+            android:shadowDy="6"
+            android:shadowRadius="3"
+            android:text="Drink Master"
+
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/admin_panel.xml]: remove line

File: app/src/main/res/layout/admin_panel.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            android:id="@+id/imageButton7"
+            android:layout_width="340dp"
+            android:layout_height="170dp"
+            android:layout_marginLeft="870dp"
+            android:layout_marginTop="215dp"
+            app:srcCompat="@drawable/restart_button" />
+
+    </FrameLayout>
+
+</LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/admin_panel.xml]: add empty line

File: app/src/main/res/layout/finished_box.xml [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+        app:srcCompat="@drawable/mocktail" />
+
+    <ImageView
+        android:id="@+id/imageView5"
+        android:layout_width="113dp"
+        android:layout_height="105dp"
+        android:layout_marginLeft="625dp"
+        android:layout_marginTop="245dp"
+        app:srcCompat="@drawable/lemon" />
 </LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/finished_box.xml]: add empty line

File: app/src/main/res/layout/ok_box.xml [hunk 4]
```diff
@@ -33,16 +33,16 @@
...
     <ImageView
         android:id="@+id/imageView5"
         android:layout_width="113dp"
         android:layout_height="105dp"
-        android:layout_marginLeft="620dp"
-        android:layout_marginTop="270dp"
+        android:layout_marginLeft="625dp"
+        android:layout_marginTop="250dp"
         app:srcCompat="@drawable/lemon" />
 </LinearLayout>
```
Comment (JoeyKraemer) [inline app/src/main/res/layout/ok_box.xml]: add empty line

**Review by JoeyKraemer — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/admin_panel.xml [hunk 5]
```diff
@@ -12,17 +12,18 @@
...
-            android:layout_marginLeft="835dp"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
             android:layout_marginTop="5dp"
             android:fontFamily="@font/share_regular"
             android:shadowColor="@color/black"
-            android:shadowDx="3"
-            android:shadowDy="3"
-            android:shadowRadius="1"
-
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/admin_panel.xml]: delete empty line

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]
Look at how the other are made this wont work

File: app/src/main/res/drawable/border.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#000000"></solid>
+    <stroke android:width="7dp" android:color="#000000"/>
+</shape>
```
Comment (JoeyKraemer) [inline app/src/main/res/drawable/border.xml]: add empty line

**Review by JoeyKraemer — APPROVED** [review]

**Review by JoeyKraemer — COMMENTED** [review]
LGTM!

File: app/src/main/res/layout/finished_box.xml [hunk 7]
```diff
@@ -20,18 +25,12 @@
...
 
-    <ImageView
-        android:id="@+id/imageView5"
-        android:layout_width="113dp"
-        android:layout_height="105dp"
-        android:layout_marginLeft="625dp"
-        android:layout_marginTop="245dp"
-        app:srcCompat="@drawable/lemon" />
 
 </LinearLayout>
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/finished_box.xml:36]: add empty line at the end

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/ok_box.xml [hunk 8]
```diff
@@ -28,12 +28,12 @@
...
-        android:layout_marginLeft="110dp"
+        android:layout_marginLeft="80dp"
         android:layout_marginTop="80dp"
         android:layout_marginBottom="75dp"
         android:baselineAligned="false"
         android:fontFamily="@font/share_regular"
-        android:text="Your drink will be prepared shortly"
+        android:text="Your drink will be prepared shortly."
         android:textColor="@color/white"
         android:textSize="45dp" />
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/ok_box.xml:39]: delete empty lines

File: app/src/main/res/layout/admin_panel.xml [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+        <ImageButton
+            android:id="@+id/imageButton7"
+            android:layout_width="340dp"
+            android:layout_height="170dp"
+            android:layout_marginLeft="870dp"
+            android:layout_marginTop="215dp"
+            app:srcCompat="@drawable/restart_button" />
+
+    </FrameLayout>
+
```
Comment (AlvesBernardo) [inline app/src/main/res/layout/admin_panel.xml]: THIS NEEDS TO BE FIXED TODAY. CLEAR IDS

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]
Constraint not connected check error message
https://developer.android.com/develop/ui/views/layout/constraint-layout

**Comment by AlvesBernardo** [comment]
Final version of Frontend finished merged to backend already.
For the main branch ONLY merge Backend do not merge FRONTNED

**Pull request merged**